### PR TITLE
added artwork sir

### PR DESCRIPTION
**Pull request by thepatienceayeni**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Excluding passwords.txt
 
-*.txt
```
Comment (pauleke65) [inline .gitignore:3]: Nice Work, removing this

File: artworks.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Monalisa
+starry night
```
Comment (pauleke65) [inline artworks.txt]: Seems incomplete, add artist names.
And a nav title too

**Review by pauleke65 — CHANGES_REQUESTED** [review]

File: artworks.txt (hunk 2 above)
Comment (thepatienceayeni) [reply artworks.txt]: What is a Nav Title

Comment (pauleke65) [reply artworks.txt]: Oh, just add something at the top of the file, saying which column is the art name and which is the artist

Like
Art Name - Artist

**Review by pauleke65 — APPROVED** [review]

**Pull request merged**